recipe/bld.bat

```diff
@@ -11,10 +11,14 @@ cmake %CMAKE_ARGS% ^
       -DCMAKE_BUILD_TYPE:STRING=Release ^
       -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% ^
       -DCMAKE_INSTALL_LIBDIR:PATH=lib ^
+      -DCCCL_ENABLE_UNSTABLE:BOOL=ON ^
```

Comment:
All these build options have been added compared to the [2.7.0 commit on conda-forge](https://github.com/conda-forge/cccl-feedstock/blob/e901cd02674a767049ee39b8fa03affd164567f1/recipe/bld.bat). Basically, they add experimental features and is probably safe to have, but looks like you pulled a later version then made changes back to 2.7.0, similarly to nvcc-impl? This might make it difficult to keep a clean added-commit history and to do the update next time

Reply:
@danpetry I re-synced with conda-forge https://github.com/conda-forge/cccl-feedstock/blob/e901cd02674a767049ee39b8fa03affd164567f1/recipe/